src/code/z_demo.c

```diff
@@ -195,6 +187,7 @@ void func_80064824(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase*
             if (sp3F != 0) {
                 func_800F6D58(0xF, 0, 0);
                 func_800753C4(globalCtx, 3);
+                if(1);
```

Comment:
This doesn't fit our style as is.
```suggestion
                if (1) {}
```

Reply:
fixed.